substrate/bin/node/runtime/src/lib.rs

```diff
@@ -784,8 +784,8 @@ impl Get<Option<BalancingConfig>> for OffchainRandomBalancing {
 			max => {
 				let seed = sp_io::offchain::random_seed();
 				let random = <u32>::decode(&mut TrailingZeroInput::new(&seed))
-					.expect("input is padded with zeroes; qed") %
-					max.saturating_add(1);
+					.expect("input is padded with zeroes; qed")
+					% max.saturating_add(1);
```

Comment:
Please keep the formatting if nothing changed.  
It otherwise just extends the review (and if needed audit) times.

Reply:
My bad, format on-save is a bit sneaky. Will revert the auto formatting.